jsonschema-core.xml

```diff
@@ -1803,24 +1815,25 @@
 ]]>
                         </artwork>
                         <postamble>
-                            Here we see that the canonical URI for that "additionalProperties"
-                            subschema is still valid, while the non-canonical URI with the fragment
-                            beginning with "#/items/$ref" now resolves to nothing.
+                            Here we see that the URI for the "additionalProperties" schema object
+                            that is relative to its resource's canonical URI is still valid,
+                            while the URI relative to the "items" schema object's URI no longer
+                            resolves to anything.
```

Comment:
I found this sentence difficult to parse. I think it would be more clear if the identifiers in question were explicit.

```suggestion
                            Here we see that `https://example.com/bar#/additionalProperties`
                            is still valid, while `https://example.com/foo#/items/additionalProperties`
                            no longer resolves to anything.
```

Reply:
@jdesrosiers I generally agree, let me think a bit on wording, I feel like something about the reasoning is lost in the change, but agree that wording closer to your suggestion will be better.